models/turbine_models/custom_models/sdxl_inference/sdxl_cmd_opts.py

```diff
@@ -170,7 +170,7 @@ def is_valid_file(arg):
 p.add_argument(
     "--vae_decomp_attn",
     type=bool,
-    default=True,
+    default=False,
```

Comment:
Didn't this give a perf regression before?

Reply:
for the sprint we switched around a few times as the attention lowerings improved, and ultimately we want to keep things in config A, but this option is here since there are a few cases I've run across where it helps (e.g. if we need vector distribution to lower SDXL config A for a target hw, and vecdist breaks just on VAE/clip, we switch to config b just for VAE so that we can still do config A on unet.)